src/System.Drawing.Common/src/System/Drawing/Gdiplus.cs

```diff
@@ -474,6 +474,10 @@ public const int
         [DllImport(ExternDll.Gdi32, SetLastError = true, ExactSpelling = true)]
         public static extern IntPtr CreateCompatibleBitmap(HandleRef hDC, int width, int height);
 
+        [DllImport(ExternDll.Gdi32, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Auto)]
```

Comment:
It's better to do a wrap for the HandleRef rather than another DllImport. 

``` C#
public static int BitBlit(HandleRef hDC, /* etc.. */)
{
   InPtr handle = BitBlit(hDC.Handle /*etc*/);
   GC.KeepAlive(hDC.Wrapper);
   return handle;
}
```